src/components/controls/selects/SearchSelect/SearchSelect.stories.tsx

```diff
@@ -82,16 +111,58 @@ const InputValueTemplate: ComponentStory<typeof SearchSelect<(typeof inputValueO
   )
 }
 
-export const InputValue = InputValueTemplate.bind({})
-InputValue.args = {
-  label: "Label",
-  options: inputValueOptions,
-}
-
-export const CustomFilter = Template.bind({})
+export const CustomFilter = DescriptionTemplate.bind({})
 CustomFilter.args = {
   ...Basic.args,
+  options: descriptionOptions,
   filter: (value, option) =>
     option.value.toLowerCase().trim().includes(value.toLowerCase().trim()) ||
     option.description.toLowerCase().trim().includes(value.toLowerCase().trim()),
 }
+CustomFilter.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement)
+  const inputField = canvas.getByRole("textbox") as HTMLInputElement
+  inputField.focus()
+  await userEvent.type(inputField, "DeScRiptiOn tHrEe", { delay: 10 })
+  await userEvent.keyboard("[Enter]")
+  await expect(inputField.value).toBe("Operator dazzling breeding")
```

Comment:
You can you `.toHaveValue()` here to avoid needed the type assertion.

```suggestion
  await expect(inputField).toHaveValue("Operator dazzling breeding")
```